msg/msg.go

```diff
@@ -0,0 +1,11 @@
+package msg
+
+import schema "gopkg.in/raintank/schema.v1"
+
+// Point represents an incoming datapoint
+// represented either as MetricData or MetricPointId2
+type Point struct {
+	Val   uint8 // 0 means use md, 1 means use point
```

Comment:
`Val` seems like the wrong term to use here. Perhaps we use `DataType` and then use
```
var MetricData = uint8(0)
var MetricPoint = uint8(1)
```
Which will make the code much more readable
```
p := msg.Point{
   DataType: msg.MetricPoint,
   Point: point,
}
```

Reply:
yep. or const even not var. but i decided to do away with this composite msg.Point type and instead use different codepaths in the input handler and index. will push soon.